sites/aviationpros.com/config/dragon-forms.js

```diff
@@ -3,6 +3,6 @@ const DragonFormsConfig = require('@endeavor-business-media/package-shared/confi
 const config = new DragonFormsConfig({ url: 'https://endeavor.dragonforms.com' });
 
 config
-  .addForm('newsletterSignup', { omedasite: 'AviationProsPrefPage' });
+  .addForm('newsletterSignup', { omedasite: 'AIRBnewpref&pk=ARTWEB' });
```

Comment:
Since `pk` is a query param it should be configured as `{ omedasite: 'AIRBnewpref', query: { pk: 'ARTWEB' } }` otherwise the `&pk=` value will be ignored -- assuming it's even needed :)

All of these will need to change, unfortunately.

Reply:
gratzi, I will get it updated